packages/@react-aria/test-utils/src/tree.ts

```diff
@@ -140,7 +159,14 @@ export class TreeTester {
         // Note that long press interactions with rows is strictly touch only for grid rows
         await triggerLongPress({element: cell, advanceTimer: this._advanceTimer, pointerOpts: {pointerType: 'touch'}});
       } else {
-        await pressElement(this.user, cell, interactionType);
+        // TODO add modifiers here? Maybe move into pressElement if we get more cases for different types of modifier keys
```

Comment:
i don't quite follow this todo

Reply:
this was more of a musing around perhaps consolidating all of these "trigger a press while holding a modifier key" into `pressElement` itself if it turns out that we have a lot of these kinds of interactions in our components